src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -580,12 +585,24 @@ public String getResourceRequestCpu() {
         return getFirstContainer().map(ContainerTemplate::getResourceRequestCpu).orElse(null);
     }
 
+    @Deprecated
```

Comment:
Why would you introduce a member which is immediately `@Deprecated`?

Reply:
right :)